Java/src/main/java/com/gildedrose/GildedRose.java

```diff
@@ -1,62 +1,89 @@
 package com.gildedrose;
 
 class GildedRose {
+
+    public static final String SULFURAS = "Sulfuras, Hand of Ragnaros";
+    public static final String CHEESE = "Aged Brie";
+    //TODO: brainstorm the naming a bit here.
+    public static final String BACKSTAGE_PASSES_TO_A_TAFKAL_80_ETC_CONCERT
+        = "Backstage passes to a TAFKAL80ETC concert";
```

Comment:
Nit: Constant name is a bit verbose, could possibly be limited to BACKSTAGE_PASSES?